<ui version="4.0" >
 <class>FileExistsDialog</class>
 <widget class="QDialog" name="FileExistsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string comment="File already exists" >File exists</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayout_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>248</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Specified file already exists, and I don't
know what to do.</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="RadioScratch" >
        <property name="text" >
         <string>Start from scratch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioContinue" >
        <property name="text" >
         <string>Continue already started job</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioUnique" >
        <property name="text" >
         <string>Make the file unique (append a number)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioCancel" >
        <property name="text" >
         <string>Cancel, do nothing</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OKbutton" >
          <property name="text" >
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OKbutton</sender>
   <signal>released()</signal>
   <receiver>FileExistsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>161</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel" >
     <x>188</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
